app/controllers/organizations_controller.rb

```diff
@@ -1,8 +1,9 @@
 class OrganizationsController < ApplicationController
   include OrganizationAuthorization
 
-  before_action :authorize_organization_addition, only: [:create]
-  before_action :set_users_github_organizations,  only: [:new, :create]
+  before_action :authorize_organization_addition,     only: [:create]
+  before_action :set_users_github_organizations,      only: [:new, :create]
+  before_action :paginate_users_github_organizations, only: [:new, :create]
```

Comment:
This is fine, since you've been using Rails callbacks for a lot of stuff in this app. Normally I'd do this in the controller action.